### ISSUE TITLE
[http-sink] Siddhi escapes first letter from Header params 

### ISSUE THREAD

**Issue by chaminda** [wso2/product-sp#28]
**Steps to Recreate:**
1. Define http sink as below.
```
@Sink(type='http',publisher.url='http://localhost:8080/testresults', headers='methodName:com.wso2.sp.test.VerifyXML', method='POST',
@map(type='json'))
define stream FooStream (message string,method String,headers String);
```
2. Monitor headers receiving at publisher.url

**Actual Result:**
the header defined above will receive as below as escaping the first letter(m) from it.
`ethodName=com.wso2.sp.test.VerifyXML`

**Expected Result:**
Defined header should be received as it is.
`methodName=com.wso2.sp.test.VerifyXML`

**Comment by Yasara123** [comment]
Header format is not in expected format. Expected header format is like "'key:value','key:value'" and in you scenario it should be  headers="'methodName:com.wso2.sp.test.VerifyXML'" had miss last apostrophe correct way should be "'methodName:com.wso2.sp.test.VerifyXML'".

**Comment by suhothayan** [comment]
@Yasara123 
Can you add this information to the HTTP docs under sink and source header properties, I can see this there. !

**Comment by Yasara123** [comment]
Fixed in f0be181aefcbf2f2762c2da7cb813904389e2e3f
